src/templates/token.tsx

```diff
@@ -172,6 +187,76 @@ const generateTokenPage = ({ secret }: { secret: string }) => {
 
           <MjmlSpacer height={values.px32} />
 
+          {/* Description card */}
+
+          <MjmlSection mjClass="bg-primary" borderRadius={values.px16}>
+            <MjmlColumn
+              mjClass="bg-primary"
+              padding={values.px30}
+              borderRadius={values.px16}
+            >
+              <MjmlImage
+                width={values.px62}
+                src={`${assetsEndpoint}/seal.png`}
+              />
+              <MjmlSpacer height={values.px10} />
+              <MjmlText
+                mjClass="font-accent text-formal-accent"
+                fontWeight={700}
+                align={aligns.center}
+                fontSize={values.px24}
+                lineHeight={values.px27}
+              >
+                Wait, what is this email?
+              </MjmlText>
+              <MjmlSpacer height={values.px10} />
+              <MjmlText
+                mjClass="font-accent text-formal-accent"
+                fontWeight={400}
+                align={aligns.center}
+                fontSize={values.px14}
+                lineHeight={values.px22}
+              >
+                <span className="break-all" style={{ color: '#ff7bed' }}>
```

Comment:
Can you use `colors.secondary` here?

Reply:
Why do you need `break-all` here? It's only two words wrapped in `break-all`

Reply:
Changed